### PR TITLE
User App

### PR DESCRIPTION
**Pull request by haowie-dev**
## Overview
Basic app module for users

## Known Issues
- None so far but will need tests

## Changelog
- Added in user app with models, migrations, serializers, views, and urls example included

## Deployment Plan
1. Restart server
2. Apply migrations

## Test Plan
1. To be filled in

### CODE REVIEW

File: apps/user/views.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from apps.user.serializer import UserSerializer
+
+
+class UserList(ListCreateAPIView, UpdateAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+    permission_classes = (IsAuthenticated,)
+
+
+class UserDetail(RetrieveUpdateDestroyAPIView):
```
Comment (roflcopterprime) [inline apps/user/views.py]: COMMO????

**Review by michaelnyu — APPROVED** [review]
looks very clean

File: BMU/settings.py [hunk 2]
```diff
@@ -1,22 +1,43 @@
...
+BASE_DIR = os.path.abspath(__file__ + '/../../../')
+
+
+def get_env_setting(setting, default=None):
+    """
+    Get the environment setting or return default value if set, otherwise return exception
+    """
+    value = os.environ.get(setting, default)
+    if default is None and value is None:
+        raise ImproperlyConfigured
```
Comment (rahulssheth1) [inline BMU/settings.py:28]: Is ImproperlyConfigured just throwing an exception?

**Review by rahulssheth1 — APPROVED** [review]
Thanks for the documented code Hao!

**Pull request merged**